### DISCUSSION TITLE
Can definePage be used anywhere?

### DISCUSSION THREAD

**Discussion by fanckush** [posva/unplugin-vue-router#113] [Questions and Help] [answered]
in the [readme](https://github.com/posva/unplugin-vue-router#sfc-route-custom-block), it says that in order to use the `<route>` block, the vue file has to be inside the routes folder.

Does this limitation apply to `definePage` as well? or can I use it on **any** `.vue` file and create a route for it on the fly regardless of where the file resides?

**Comment by posva** [comment, marked as answer]
`definePage()` must be used within a page component, other files are not parsed nor associated with routes

**Reply by fanckush** [reply]
thanks
